lms/static/scripts/frontend_apps/components/AppRoot.tsx

```diff
@@ -41,7 +41,7 @@ export default function AppRoot({ initialConfig, services }: AppRootProps) {
               <FilePickerApp />
             </DataLoader>
           </Route>
-          <Route path="/dashboard/organization/:organizationId/assignment/:assignmentId">
+          <Route path="/dashboard/organization/:organizationId" nest>
```

Comment:
Leaving here only the part that will be common to all routes. We could leave just `/dashboard`, but I reckon `/organization/:organizationId` will be also needed for other views.